stdlib-candidate/std-rfc/mod.nu

```diff
@@ -0,0 +1,2 @@
+export module aggregate
```

Comment:
is this line expected? i don't see any mention to this module anywhere :open_mouth:

Reply:
It's in the std-rfc `stdlib-candidate/std-rfc/aggregate/mod.nu`

Reply:
yeah maybe it's legit overall, just does not look like it belongs here :wink: